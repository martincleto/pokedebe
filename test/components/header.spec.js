import Utils from 'react-addons-test-utils'

describe('Header', function() {

  it('can render without error', function() {
    // const renderer = ReactTestUtils.createRenderer();
    // renderer.render(<MyComponent />);
    // const result = renderer.getRenderOutput();
    //
    // expect(result.type).toBe('Hea');
    // expect(result.props.children).toEqual([
    //   <span className="heading">Title</span>,
    //   <Subcomponent foo="bar" />
    // ]);
  });
})
