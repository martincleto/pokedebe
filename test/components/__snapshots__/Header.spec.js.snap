// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render without error 1`] = `
<header>
  <h1
    className="heading heading--level-1"
  >
    Pok√©debe
  </h1>
</header>
`;
